codelabs/Utopia Wholesale Codelab/2-holder.md

```diff
@@ -225,12 +225,22 @@ if (DigitalCredentials.Default.available) {
 
 The Holder app also needs to add the Verifier (Reader) certificate to its trust list. This ensures that the Holder can recognize and trust the Verifier during credential sharing. The Verifier's certificate can be downloaded from the Multipaz Verifier [website](https://verifier.multipaz.org/identityreaderbackend/). Below is the code snippet demonstrating how to add the Verifier's certificate to the trust list:
 ```
-   addTrustPoint(
+        addTrustPoint(
```

Comment:
Could we include the other trustpoints for the Multipaz test app in the Get Started guide? Even though the codelab works with the Reader App, it would be great if all Multipaz apps could be used with the Codelab and Get Started docs. Here’s the link for reference: https://developer.multipaz.org/docs/getting-started/holder#%EF%B8%8F-reader-trust